plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/lsp/PatchedRemoteEndpoint.java

```diff
@@ -167,7 +167,7 @@ protected void handleRequest(RequestMessage requestMessage) {
 							t);
 				}
 				responseMessage = createErrorResponseMessage(requestMessage, errorObject);
-				LOG.debug("Server errors: " + messageId + " / " + requestMessage.getMethod());
+				LOG.error("Server errors: " + messageId + " / " + requestMessage.getMethod());
```

Comment:
Exceptions should result in an error to be logged, I figured. However, the case above (lines 158ff) is a cancellation exception and not an error. Anyhow, I wonder whether cancellation exceptions should reach this location or should be caught some before.